h/schemas/admin_group.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import colander
+from deform.widget import SelectWidget, TextInputWidget
+
+from h import i18n
+from h import validators
+from h.models.group import (
+    GROUP_NAME_MIN_LENGTH,
+    GROUP_NAME_MAX_LENGTH,
+    GROUP_DESCRIPTION_MAX_LENGTH
+)
+from h.schemas.base import CSRFSchema
+
+_ = i18n.TranslationString
+
+VALID_GROUP_TYPES = (
+    (u'private', _('Private')),
+    (u'restricted', _('Restricted')),
+    (u'open', _('Open')),
+)
+
+
+class CreateAdminGroupSchema(CSRFSchema):
+
+    group_type = colander.SchemaNode(
+        colander.String(),
+        title=_('Group Type'),
+        widget=SelectWidget(values=((u'', _('Select')),) + VALID_GROUP_TYPES)
+    )
+
+    name = colander.SchemaNode(
+        colander.String(),
+        title=_('Group Name'),
+        validator=validators.Length(min=GROUP_NAME_MIN_LENGTH,
+                                    max=GROUP_NAME_MAX_LENGTH),
+    )
+
+    authority = colander.SchemaNode(
+        colander.String(),
+        title=_('Authority'),
+        description=_("The group's authority"),
```

Comment:
None of the templates available for forms/deform ever render `description`...I didn't get to addressing this in this PR, but having to hover over a tiny tooltip to get any further info about a field (the `hint`) is tough on usability.